packages/amplication-design-system/src/components/HorizontalRule/HorizontalRule.tsx

```diff
@@ -1,9 +1,20 @@
 import React from "react";
-
+import classNames from "classnames";
 import "./HorizontalRule.scss";
 
 const CLASS_NAME = "amp-horizontal-rule";
 
-export function HorizontalRule() {
-  return <hr className={CLASS_NAME} />;
+export enum EnumHorizontalRuleStyle {
+  Black5 = "black5",
```

Comment:
it's risky. if someone will change the css, it will need to change the names. maybe will add a comment for that

Reply:
haha... i agree, but we talked about it and you suggested it :-)